debian/changelog

```diff
@@ -1,3 +1,161 @@
+  * tests: move the plainbox test to the integration suite (#1642)
+  * lxd: fix the push in container builds (#1644)
+  * tests: use the common base handler on the fake snapd server (#1724)
+  * tests: split the integration autopkgtests (#1716)
+  * tests: in autopkgtests, use a tempdir in home, not in the tmpfs (#1657)
+  * recording: record information from the image in container builds  (#1633)
+  * recording: pass the build info flag to the container (#1736)
+
+  [ Christian Dywan ]
+  * lxd: mount project folder via sshfs in case of a remote (#1302)
+  * lxd: Only pass target arch if specified explicitly
+  * repo: friendly, helpful error for unsupported distros (#1586)
+  * lxd: instructions for /etc/sub{u,g}id after failed start (#1553)
+  * lxd: pass SNAPCRAFT_PARTS_URI through into container (#1585)
+  * lxd: use SUDO_UID for ID mapping (#1588)
+  * lxd: don't inject local snaps on a different arch (#1577)
+  * lxd: don't re-inject the same snaps (#1568)
+  * lxd: split container classes into different files (#1627)
+  * cli: update parts cache in the container (#1546)
+  * lifecycle: clean after deleting container (#1587)
+  * lxd: snapcraft refresh in containers (#1412)
+  * lxd: distinguish argless clean from clean -s pull (#1655)
+  * lxd: refresh remote container (#1739)
+  * cli: pass remote from container_config to clean method (#1737)
+
+  [ Mark Lee ]
+  * project_loader: quote more environment variable values (#1578)
+
+  [ James Beedy ]
+  * ruby plugin: new plugin
+
+  [ Aleix Pol ]
+  * repo: return a proper value in DummyRepo
+  * common: do not fail over on empty or faulty lines in os-release
+
+  [ Jeff Dickey ]
+  * ci: install git in Dockerfile for '{version: git}' usage (#1575)
+
+  [ Michael Vogt ]
+  * meta: add and adapter property for apps
+
+  [ Chris Ratliff ]
+  * catkin plugin: allow ROS_MASTER_URI change (#1572)
+
+  [ Rakesh Singh ]
+  * dotnet plugin: new plugin (#1574)
+
+  [ Martin Wimpress ]
+  * nodejs plugin: update default node engine to 6.11.4 (#1589)
+
+  [ Colin Watson ]
+  * options: fix core-dynamic-linker on ppc64el/s390x (#1600)
+
+  [ Paolo Pisati ]
+  * kbuild plugin: if the parts build dir already contains a .config file, return immediately (#1606)
+  * cross compilation: enable cross compilation of i386 kernel on x86-64 … (#1613)
+
+  [ Jonathan Cave ]
+  * plainbox-provider plugin: init PROVIDERPATH (#1611)
+
+  [ Carlo Lobrano ]
+  * store: fix StoreReleaseError format for BAD REQUEST error (#1599)
+  * sources: get svn revision with --show-item flag (#1648)
+  * Removed dependency on VERSION_ID in os-release (#1623)
+
+  [ Alfonso Sanchez Beato ]
+  * kernel plugin: use latest stable core snap (#1624)
+
+  [ Nathan Haines ]
+  * ruby plugin: new stable release 2.4.2 (#1645)
+
+  [ Marius Gripsgard ]
+  * sources: workaround for ZipFile.extractall not preserving permissions (#1723)
+
+ -- Sergio Schvezov <sergio.schvezov@canonical.com>  Wed, 01 Nov 2017 19:41:33 +0000
```

Comment:
Worth updating this?

Reply:
wow, let's leave this as a reminder of how long it took!